frontend/server/src/Controllers/Problem.php

```diff
@@ -3873,20 +3911,56 @@ public static function getProblemEditDetailsForSmarty(
             'STATUS_SUCCESS' => \OmegaUp\Translations::getInstance()->get(
                 'problemEditUpdatedSuccessfully'
             ),
+            'payload' => [
+                'languages' => \OmegaUp\Controllers\Run::DEFAULT_LANGUAGES,
+                'validLanguages' => $validLanguages,
+                'validatorsTypes' => $validatorsTypes,
+            ]
```

Comment:
nit:

```suggestion
            ],
```

dentro de un contexto de lista, siempre hay que poner coma después de cada elemento.

Reply:
listo